hack/go-build.sh

```diff
@@ -9,8 +9,8 @@ GLDFLAGS=${GLDFLAGS:-}
 
 eval $(go env | grep -e "GOHOSTOS" -e "GOHOSTARCH")
 
-GOOS=${GOOS:-${GOHOSTOS}}
-GOARCH=${GOACH:-${GOHOSTARCH}}
+: "${GOOS:=${GOHOSTOS}}"
```

Comment:
I don't think you want the quotes. This should be sufficient:

```sh
: ${GOOS:=${GOHOSTOS}}
```

Reply:
Maybe you are running shellcheck?

https://github.com/koalaman/shellcheck/wiki/SC2223

Its advice is to quote. So be it.